change/@ni-nimble-components-9d33a263-abcb-4b8e-a5f1-1ab02c9b4974.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The [diffs in the current Chromatic build](https://www.chromatic.com/test?appId=60e89457a987cf003efc0a5b&id=65bab7b6817624700576dac2) are concerning, with several number-text columns no longer rendering data. Can you take a look?

Reply:
Well, this is super annoying. Chromatic's versions of Chrome and Firefox are too old (9/22 and 8/22 respectively) to support the options I'm trying to use. Historically, Chromatic has [updated their browsers](https://www.chromatic.com/docs/infrastructure-release-notes/) about once every 6 or 12 months, and with their last update in February 2023, we should be due for another any time now.

Though it looks wrong, I'm inclined to say we just live with this small blind spot in our visual testing until they publish another update. The Chromatic tests are most useful for catching minor visual changes. We're probably not going to be doing any new development on the number-text column that would result in those kinds of subtle visual changes. Problems resulting in more significant visual changes are likely to be caught during dev via Storybook or functional tests.

Reply:
My vote would be to put this on the backburner until Chromatic publishes a browser update. For me, fixing the tech debt isn't worth regressing our test suite, even if it is a temporary regression that might not allow any bugs to slip through in the near term.

Another idea would be to keep both the old and new code around for now, and fall back to the old code if we can detect that the new code isn't supported. But I imagine that would be messy and it would still result in us needing a tech debt issue to track removing the old code.

Happy to discuss with the team at our meeting today.

Reply:
Chromatic updated, but I'm now seeing improper rendering in the Chromatic test in Safari. I can't reproduce with any version of WebKit I try. According to the documentation, the new version of Chromatic Capture Cloud uses Safari 17, but when I dumped the value of `navigator.userAgent` from the test, it indicates that it's version 17.4 (which is weird, because 17.4 is currently only in beta):
![image](https://github.com/ni/nimble/assets/7282195/d2ba3ee0-37fa-46f1-9343-1963e8eb229c)

Reply:
Jesse tested Safari Tech Preview 17.4, and it still did not reproduce the issue. It's unclear what version/build of Safari is being used that seems to uniquely have this problem. Jesse stated that he was fine living with the spurious diff, and I think I agree.